### PR TITLE
Initial working pages for path and network

### PR DESCRIPTION
**Pull request by timadair**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
-# relationship-graph-examples
\ No newline at end of file
+#Senzing Relationship Graph Examples
```
Comment (arawinters) [inline README.md]: hash symbol space.

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+
+##A Quick Example
+An example file is provided to show what a finished graph looks like.
+1. Open src/network/index.html in a browser
+2. Click the **Choose File** button in the **Find Network Raw JSON File** area.
+3. Select `example-network-data.json`
+4. Click the **Load** button in the **Find Network Raw JSON File** area.
+5. Uncheck the **Show Match Keys** checkbox to hide the link labels.  Link details can be seen by mousing over the link.
+
+## Graphing your Project's Relationships
```
Comment (arawinters) [inline README.md]: I'd move this up, overview usually proceeds bullet points/toc.

File: src/network/index.html [hunk 3]
```diff
@@ -0,0 +1,626 @@
+<!DOCTYPE html>
```
Comment (arawinters) [inline src/network/index.html]: I'd add README.md's to each of these directories, or relocate/copy the content specific to the example from the main README.md so when browsing the project on github the subdir will get the explanation.

File: src/network/index.html [hunk 4]
```diff
@@ -0,0 +1,626 @@
...
+<meta charset="utf-8">
+
+<style>
```
Comment (arawinters) [inline src/network/index.html]: is the styling particularly relevant to the example code? like will the example fall apart without it? if not might want to move it to a separate file to keep the example as easy to follow as possible.

File: src/network/index.html [hunk 5]
```diff
@@ -0,0 +1,626 @@
...
+    default: {
+      shape: "M343.771 124.343c43.886-29.257 102.4-43.886 175.543-43.886 87.771 0 160.914 21.943 226.743 65.829 58.514 43.886 87.771 109.714 87.771 190.171 0 51.2-14.629 95.086-36.571 131.657-14.629 21.943-43.886 51.2-87.771 80.457l-51.2 36.571c-21.943 14.629-36.571 36.571-43.886 58.514 0 14.629-7.314 36.571-7.314 73.143h-160.914c0-65.829 7.314-117.029 21.943-138.971 7.314-29.257 36.571-58.514 80.457-87.771l36.571-36.571c14.629-7.314 29.257-21.943 36.571-36.571 14.629-21.943 21.943-43.886 21.943-73.143s-7.314-58.514-29.257-80.457c-14.629-21.943-43.886-36.571-87.771-36.571s-80.457 14.629-95.086 43.886c-21.943 29.257-36.571 65.829-36.571 95.086h-168.229c0-109.714 43.886-190.171 117.029-241.371zM446.171 797.257h175.543v175.543h-175.543v-175.543z",
+    }
+  };
+
+  const port = '2080';
+  const svgWidth = 2000, svgHeight = 960; // 4x the size of path because networks involve more nodes.
+  let entityInputCount = 2;
+
+  document.getElementById('btnLoadFile').addEventListener('click', loadFile);
```
Comment (arawinters) [inline src/network/index.html]: add comments on what these are doing.

File: src/network/index.html [hunk 6]
```diff
@@ -0,0 +1,626 @@
...
+    inputs.appendChild(newInputLabel);
+    inputs.appendChild(newInput);
+    inputs.appendChild(newInputBreak);
+  }
+
+  /**
+   * Retrieves the networks around and between all input entities and initializes the Force Layout Diagram
+   * Uses the Senzing REST API
+   */
+  async function loadRest() {
```
Comment (arawinters) [inline src/network/index.html]: mention async in the summary. some users will not know what this is.

File: src/network/index.html [hunk 7]
```diff
@@ -0,0 +1,626 @@
...
+  }
+
+  let node,  // The D3 selector for elements with class of .node
+      link,  // The D3 selector for elements with class of .link
+      force; // The D3 force simulation
+
+  /**
+   * Initializes the Force Layout Diagram.
+   * @param senzingData
+   */
```
Comment (arawinters) [inline src/network/index.html]: no idea what @param is, add @see or @link to OAS spec if that's what that is.

File: src/network/index.html (hunk 4 above)
Comment (timadair) [reply src/network/index.html]: No, the stying isn't necessary for understanding the examples, but I move them or I can add some comments that would help someone new to D3 understand how it works and how we use them.  Any preference between those two?

**Pull request merged**